REPOSITORIES.md

```diff
@@ -0,0 +1,62 @@
+# Repositories
+
+This document lists all repositories, sorted by their scope, within [Dragonfly](https://github.com/dragonflyoss).
+
+You can read more about the scope and lifecycle of repositories in [REPOSITORY-GUIDELINES.md](https://github.com/dragonflyoss/community/blob/main/REPOSITORY-GUIDELINES.md).
+
+## `Core` Repositories
+
+Learn more about the `Core` scope repositories in [REPOSITORY-GUIDELINES.md](./REPOSITORY-GUIDELINES.md#core-repositories).
+
+<!-- markdownlint-disable -->
+| Name                                                               | Description                                                                               |
+| :----------------------------------------------------------------- | :---------------------------------------------------------------------------------------- |
+| [.github](https://github.com/dragonflyoss/.github)                 | The default community organization-level files for Dragonfly.                             |
+| [api](https://github.com/dragonflyoss/api)                         | Canonical location of the Dragonfly API definition files.                                 |
+| [client](https://github.com/dragonflyoss/client)                   | The Dragonfly client, providing a command-line interface for Dragonfly.                   |
+| [community](https://github.com/dragonflyoss/community)             | The Dragonfly community repository, providing resources and information for contributors. |
+| [console](https://github.com/dragonflyoss/console)                 | Dragonfly Console                                                                         |
+| [d7y.io](https://github.com/dragonflyoss/d7y.io)                   | The official site for Dragonfly, built with Docusaurus.                                   |
+| [design](https://github.com/dragonflyoss/design)                   | Dragonfly Design Documents                                                                |
+| [dragonfly](https://github.com/dragonflyoss/dragonfly)             | The core Dragonfly repository, containing the main codebase and documentation.            |
+| [helm-charts](https://github.com/dragonflyoss/helm-charts)         | The Helm charts for deploying Dragonfly.                                                  |
+| [nydus](https://github.com/dragonflyoss/nydus)                     | Nydus provides fast, secure and easy access to container images.                          |
```

Comment:
` Nydus is a sub-project, provides fast, secure and easy access to container images. `